BigBlueButton-RLP/Constants/BBBURL.swift

```diff
@@ -6,7 +6,7 @@ enum BBBURL {
     static var baseUrl: URL? {
         return AppEnvironment.isDebugMode ?
         // Debug
-        URL(string: "https://bbbtest.zdv.uni-mainz.de/b/") :
+        URL(string: "https://droplet-1146.meetbbb.com") :
```

Comment:
Does it still work with uni-mainz.de?

Reply:
Unfortunately, after detailed testing, screen sharing is not working on uni-mainz.de installation. Im not sure why is this the case when its working normally on Blindside installation. We need to debug University installation with Tiago or someone else from BBB.